<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderEditor</class>
 <widget class="QMainWindow" name="ShaderEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="shaderType_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SHADER TYPE :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="shaderType_Combo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Vertex</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fragment</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="vertexShader_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>571</width>
      <height>421</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveShader_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFile_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SHADER EDITOR</string>
    </property>
   </widget>
   <widget class="QLabel" name="shaderType_Label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1 - Set the desired shader type</string>
    </property>
   </widget>
   <widget class="QLabel" name="shaderType_Label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2 - Set the shader code </string>
    </property>
   </widget>
   <widget class="QPushButton" name="setCurrentScene_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set to current scene</string>
    </property>
   </widget>
   <widget class="QComboBox" name="viewTarget_combo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Left</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Top</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Perspective</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuShader_Editor">
    <property name="title">
     <string>Shader Editor</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLinker">
    <property name="title">
     <string>Linker</string>
    </property>
   </widget>
   <addaction name="menuShader_Editor"/>
   <addaction name="menuLinker"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
